qemu/tests/cfg/balloon_small_size.cfg

```diff
@@ -0,0 +1,12 @@
+- balloon_small_size:
+    virt_test_type = qemu
+    no Win2000
+    only Windows
```

Comment:
Can you show me where the developer said they will realize it ?  Our case contains linux and window guest for now, and set different target size for them.  I think currently we better match the case.  If developer realize what you said in future, we can update then. By the way, our case need to reboot guest and recheck, which is not seen in your code. Thanks